### DISCUSSION TITLE
Merging sva + css styles

### DISCUSSION THREAD

**Discussion by adbutterfield** [chakra-ui/panda#2650] [General]
I see there's an example for cva + css in [the docs here](https://panda-css.com/docs/concepts/merging-styles#merging-cva--css-styles). But there's no example for sva + css.

I just tired in a component to do like this:
```ts
const { slot1, slot2 } = componentStyle.raw({ prop1, prop2 });
```

And it seems to work the same way as merging cva + css. 

I just wanted to double check that this is good pattern?

**Comment by segunadebayo** [comment]
@adbutterfield, that's correct.

Feel free to contribute this to the docs when you have time.

**Reply by adbutterfield** [reply]
Thanks!!! Will do.
